### PR TITLE
Remove some references to Docker

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan looks like you need a rebased.  Changes LGTM.  I just want to be sure to keep at least a few actual Dockerfiles in our test subdirectories to make sure it always gets picked up automagically during a build.

**Comment by rhatdan** [comment]
There are lots of Dockerfiles down there.  Many more then Containerfiles.  I just figured rather then tell the PR it does not need tests, that I would add a few more Containerfile tests.

**Comment by TomSweeneyRedHat** [comment]
Tests still are not happy @rhatdan

File: cmd/buildah/bud.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ type budOptions struct {
...
 func init() {
 	budDescription := `
-  Builds an OCI image using instructions in one or more Dockerfiles.
+  Builds an OCI image using instructions in one or more Containerfiles.
 
   If no arguments are specified, Buildah will use the current working directory
-  as the build context and look for a Dockerfile. The build fails if no
-  Dockerfile is present.`
+  as the build context and look for a Containerfile. The build fails if no
+  Containerfile is present.`
```
Comment (edsantiago) [inline cmd/buildah/bud.go:36]: Should probably be 'no Containerfile nor Dockerfile' ?

**Review by edsantiago — COMMENTED** [review]
LGTM at a quick glance; have not dived deep. Some suggestions but not blockers.

**Comment by ashley-cui** [comment]
LGTM

**Pull request merged**